install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -0,0 +1,139 @@
+# Kubecost Aggregator
+
+{% hint style="info" %}
+Kubecost Aggregator is a beta feature.
+{% endhint %}
+
+Aggregator is a new, experimental backend for Kubecost. It is used in a [Federated ETL](/install-and-configure/install/multi-cluster/federated-etl/federated-etl.md) configuration without Thanos, replacing the [Federator](/install-and-configure/install/multi-cluster/federated-etl/federated-etl.md#other-components) component.
+Aggregator serves a critical subset of Kubecost APIs, but will eventually be the default model for Kubecost and serve all APIs. Currently, Aggregator all major monitoring and savings APIs, and also budgets and reporting.
+
+{% hint style="info" %}
+Existing documentation for Kubecost APIs will use endpoints for non-Aggregator environments unless otherwise specified, but will still be compatible after configuring Aggregator.
+{% endhint %}
+
+Aggregator is designed to accommodate queries of large-scale datasets, including reduced load times and potential errors in the UI. It is not designed to introduce new functionality; it is meant to improve functionality at scale.
+
+Aggregator is free for all users to configure, and is always able to be rolled back.
+
+## Configuring Aggregator
+
+### Prerequisites
+
+* Aggregator can only be configured in a Federated ETL environment
+* Must be using v1.107.0 of Kubecost or newer
+* Your *values.yaml* file must have set `kubecostDeployment.queryServiceReplicas` to its default value `0`. 
+
+### Tutorial
+
+Select from one of the two templates below and save the content as *aggregator.yaml*. This will be your configuration template required to set up Aggregator.
+
+Basic configuration:
+
+```
+kubecostAggregator:
+  replicas: 1
+  enabled: true
+  cloudCost:
+    enabled: true
+federatedETL:
+  federatedCluster: true
+kubecostModel:
+  containerStatsEnabled: true
+  federatedStorageConfigSecret: federated-store
+kubecostProductConfigs:
+  clusterName: YOUR_CLUSTER_NAME
+  cloudIntegrationSecret: cloud-integration
+  productKey:
+    enabled: true
+    key: YOUR_KEY
+prometheus:
+  server:
+    global:
+      external_labels:
+        cluster_id: YOUR_CLUSTER_NAME
+# when using managed identity/irsa, set the service account accordingly:
+serviceAccount:
+  create: false
+  name: kubecost-irsa-sa
+```
+
+Advanced configuration (for larger deployments):
+
+```
+kubecostAggregator:
+  replicas: 1
+  enabled: true
+  cloudCost:
+    enabled: true
+  env:
+    # governs parallelism of derivation step
+    # more threads speeds derivation, but requires significantly more 
+    # disk space for WAL storage
+    # default: 6
+    NUM_DERIVER_THREADS: 6
+    # log level
+    # default: info
+    LOG_LEVEL: debug

```

Comment:
Do we want to recommend `debug` when we say default is `info`?

Reply:
Yeah I might leave this alone completely or set in in the doc to `info` and then list in the comment what the options are (`debug'`, `info`, etc)

Reply:
agreed on info. debug adds needless system load and can drive up log system costs if it is paid